src/notifications/__init__.py

```
@@ -0,0 +1,2 @@
+from notifications.interfaces import EmailSenderInterface
```

Ruff (0.11.9)

`notifications.interfaces.EmailSenderInterface` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)